app/views/foreman/unattended/finish-katello.erb

```diff
@@ -11,6 +11,11 @@ oses:
 - Fedora 19
 - Fedora 20
 %>
+<% if @host.subnet.respond_to?(:dhcp_boot_mode?) -%>
+<%= snippet 'kickstart_networking_setup' %>
+service network restart
```

Comment:
@jlsherrill Finish script's aren't used during kickstart (despite the snippet's name that's being included), it happens via SSH.  It should be safe to do if networking info is correct